src/meili/traffic_segment_matcher.cc

```diff
@@ -10,35 +10,10 @@
 
 namespace {
 
-  boost::property_tree::ptree parse_json(const std::string& json, const std::unordered_set<std::string> customizable, boost::property_tree::ptree& match_config) {
+  boost::property_tree::ptree parse_json(const std::string& json) {
     boost::property_tree::ptree request;
     try { std::stringstream stream(json); boost::property_tree::read_json(stream, request); }
     catch (...) { throw std::runtime_error("Couln't parse json input"); }
-
-    if(customizable.empty())
-      return request;
-
-    auto match_options = request.get_child_optional("match_options");
```

Comment:
This commit renames the `match_options` object to `trace_options` for consistency with the publicly documented `trace_*` requests, since this code was duplicated both here and in thor's worker. 

I couldn't find any public docs on traffic matching, so I don't know the impact of such a breaking json attribute change. One alternative could be to handle [multiple options here](https://github.com/valhalla/valhalla/pull/1260/files#diff-329096d7bcd764e3019caedc27da61d6R107).